client/zeth/core/zksnark.py

```diff
@@ -158,17 +156,16 @@ def proof_to_proto(
         return extproof_proto
 
     @staticmethod
-    def mixer_proof_parameters(extproof: GenericProof) -> List[List[Any]]:
+    def proof_to_evm_parameters(extproof: GenericProof) -> List[List[int]]:
```

Comment:
`proof_to_evm_parameters` -> `proof_to_evm_words`?

Reply:
(same for functions below)

Reply:
I'm not attached to this encoding name, but couldn't think of a better one.

These fucntions generate the set of parameters to pass to the contract call.  By coincidence it happens to be all evm_words here, but as we move to use more structs the types here could change (in fact, a later PR flattens this to `List[int]`, but I expect it will eventually be deeper and we may want to use `bytes` etc.

I considered something like `eth_abi`, but that's the result of later encoding this to a binary string.  Other options could be `contract_parameters` or `web3_parameters` or something?

Reply:
> These fucntions generate the set of parameters to pass to the contract call.
> Other options could be contract_parameters or web3_parameters or something?

Why not naming the function: `proof_obj_to_contract_parameters` then? (same for other related functions)
The EVM is a stack-based VM processing (popping/pushing) 256-bit long words. Hence, if the goal is to convert objects (like `extended_proof`) to contract parameters, I find the current name slightly misleading, as I'd expect everything to be systematically converted to 256-bit long words instead

Reply:
> > Other options could be contract_parameters or web3_parameters or something?

> Why not naming the function: proof_obj_to_contract_parameters then?

Renamed the encoding to `contract_parameters`

Reply:
Thanks @dtebbs